ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs

```diff
@@ -20,5 +45,281 @@ public override string Name
                 return "AsyncLocalStorage";
             }
         }
+
+        /// <summary>
+        /// Given an array of keys, this returns a map of (key, value) pairs for the keys found, and
+        /// (key, null) for the keys that haven't been found.
+        /// </summary>
+        /// <param name="keys">Array of key values</param>
+        /// <param name="callback">Callback.</param>
+        [ReactMethod]
+        public void multiGet(string[] keys, ICallback callback)
+        {
+            Debug.Assert(keys != null);
+            Debug.Assert(callback != null);
+
+            var result = new JArray();
+            foreach (var key in keys)
+            {
+                result.Add(new JArray
+                {
+                    key,
+                    getTokenFromContainer(key),
+                });
+            }
+            callback.Invoke(result);
+        }
+
+        /// <summary>
+        /// Inserts multiple (key, value) pairs. The insertion will replace conflicting (key, value) pairs.
+        /// </summary>
+        /// <param name="keyValueArray">Array of key values</param>
+        /// <param name="callback">Callback.</param>
+        [ReactMethod]
+        public void multiSet(JArray keyValueArray, ICallback callback)
+        {
+            Debug.Assert(keyValueArray != null);
```

Comment:
If you look at the Android code base, it seems that the keyValueArray can be null:
[https://github.com/CatalystCode/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/storage/AsyncStorageModule.java#L153](https://github.com/CatalystCode/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/storage/AsyncStorageModule.java#L153)